server/test/setup.js

```diff
@@ -0,0 +1,15 @@
+const sinon = require('sinon');
+const logger = require('@financial-times/n-logger').default;
+
+let loggerSandbox;
+
+beforeEach(() => {
+	loggerSandbox = sinon.sandbox.create();
+	[ 'info', 'warn', 'error' ].map(logLevel => {
+		loggerSandbox.stub(logger, logLevel);
```

Comment:
Would this mask error logs for genuine test fails?

Perhaps it would be better to have the stub replace the call with a call to console.<logLevel>?

Reply:
Excellent point. I'll change that.